src/import/FSHImporter.ts

```diff
@@ -775,26 +775,30 @@ export class FSHImporter extends FSHVisitor {
   }
 
   visitMixins(ctx: pc.MixinsContext): string[] {
+    let mixins: string[];
     if (ctx.COMMA_DELIMITED_SEQUENCES()) {
-      let mixins = ctx
+      logger.warn('Using "," to list mixins is deprecated. Please use "and" when listing mixins.', {
```

Comment:
But this will be a moot point very soon when we move to `insert` rules.  In fact, we might want to remove this warning when we move to `insert` rules so we don't confuse the user by saying "don't use Mixins keyword" and "don't use commas with the Mixins keyword" at the same time.